toolbox/certbot/renew_domain_with_certbot.sh

```diff
@@ -30,12 +30,12 @@ rm $DOMAIN.pem
 
 docker pull certbot/dns-route53:v2.6.0
 
-docker run -it --rm --name certbot \
+docker run --rm --name certbot \
 -e AWS_ACCESS_KEY_ID=$(echo $AWS_ACCESS_KEY_ID | tr -d '\r' ) \
 -e AWS_SECRET_ACCESS_KEY=$(echo $AWS_SECRET_ACCESS_KEY | tr -d '\r' ) \
 -v "/etc/letsencrypt:/etc/letsencrypt" \
 -v "/var/lib/letsencrypt:/var/lib/letsencrypt" \
-certbot/dns-route53 certonly --dns-route53 --force-renewal -d $DOMAIN
+certbot/dns-route53 certonly -n --dns-route53 -d $DOMAIN
```

Comment:
I've removed `--force-renewal` which @mddilley had identified as not what we want since we want to be able to run this cron job frequently and do want to respect `certbot`'s opinion about what needs to be renewed and what does not.